agent/bench-scripts/pbench-fio

```diff
@@ -426,14 +426,16 @@ function fio_install() {
 # install python-pandas on the controller: fiologparser_hist.py needs it
 function pandas_install() {
 	if [ "$postprocess_only" = "n" ]; then
-		if check_install_rpm python-pandas; then
+	    if check_install_rpm python2-pandas; then
+			debug_log "[$script_name]python2-pandas is installed"
+		elif check_install_rpm python-pandas; then
 			debug_log "[$script_name]python-pandas is installed"
 		else
 			debug_log "[$script_name]python-pandas installation failed, exiting."
                     	debug_log "[$script_name]On RHEL, python-pandas is in the EPEL repo."
                     	debug_log "[$script_name]See https://fedoraproject.org/wiki/EPEL for details."
 			exit 1
-		fi
+		fi	
```

Comment:
Should also only be indented to the same level as the `if`, and trailing whitespace removed.